puppet/modules/quickstack/manifests/neutron/controller.pp

```diff
@@ -176,118 +176,118 @@
+    cinder_nfs_shares                                                      => $cinder_nfs_shares,
+    cinder_nfs_mount_options                                               => $cinder_nfs_mount_options,
+    cinder_san_ip                                                          => $cinder_san_ip,
+    cinder_san_login                                                       => $cinder_san_login,
+    cinder_san_password                                                    => $cinder_san_password,
+    cinder_san_thin_provision                                              => $cinder_san_thin_provision,
+    cinder_eqlx_group_name                                                 => $cinder_eqlx_group_name,
+    cinder_eqlx_pool                                                       => $cinder_eqlx_pool,
+    cinder_eqlx_use_chap                                                   => $cinder_eqlx_use_chap,
+    cinder_eqlx_chap_login                                                 => $cinder_eqlx_chap_login,
+    cinder_eqlx_chap_password                                              => $cinder_eqlx_chap_password,
+    cinder_netapp_hostname                                                 => $cinder_netapp_hostname,
+    cinder_netapp_login                                                    => $cinder_netapp_login,
+    cinder_netapp_password                                                 => $cinder_netapp_password,
+    cinder_netapp_server_port                                              => $cinder_netapp_server_port,
+    cinder_netapp_storage_family                                           => $cinder_netapp_storage_family,
+    cinder_netapp_transport_type                                           => $cinder_netapp_transport_type,
+    cinder_netapp_storage_protocol                                         => $cinder_netapp_storage_protocol,
+    cinder_netapp_nfs_shares                                               => $cinder_netapp_nfs_shares,
+    cinder_netapp_nfs_shares_config                                        => $cinder_netapp_nfs_shares_config,
+    cinder_netapp_volume_list                                              => $cinder_netapp_volume_list,
+    cinder_netapp_vfiler                                                   => $cinder_netapp_vfiler,
+    cinder_netapp_vserver                                                  => $cinder_netapp_vserver,
+    cinder_netapp_controller_ips                                           => $cinder_netapp_controller_ips,
+    cinder_netapp_sa_password                                              => $cinder_netapp_sa_password,
+    cinder_netapp_storage_pools                                            => $cinder_netapp_storage_pools,
+    cinder_rbd_pool                                                        => $cinder_rbd_pool,
+    cinder_rbd_ceph_conf                                                   => $cinder_rbd_ceph_conf,
```

Comment:
I was fixing whitespace using lint and checked in unwanted changes. Fixed it now